docs/user-guide/scheduled-jobs.md

```diff
@@ -0,0 +1,155 @@
+---
+assignees:
+- erictune
+- soltysh
+- janetkuo
+
+---
+
+* TOC
+{:toc}
+
+## What is a _Scheduled Job_?
+
+A _Scheduled Job_ manages time based [Jobs](/docs/user-guide/jobs/), namely:
+
+* Once at a specified point in time
+* Repeatedly at a specified point in time
+
+One ScheduledJob object is like one line of a _crontab_ (cron table) file. It runs a job periodically
+on a given schedule, written in [Cron](https://en.wikipedia.org/wiki/Cron) format.
+
+A typical use case is:
+
+* Schedule a job execution at a given point in time.
+* Create a periodic job, e.g. database backup, sending emails.
+
+### Prerequisites
+
+You need a working Kubernetes cluster at version >= 1.4, with batch/v2alpha1 API turned on by passing
+`--runtime-config=batch/v2alpha1` while brining up the API server (see [Turn on or off an API version
+for your cluster](/docs/admin/cluster-management/#turn-on-or-off-an-api-version-for-your-cluster) for
+more). You cannot use Scheduled Jobs on a hosted Kubernetes provider that has disabled alpha resources.
+
+## Creating a Scheduled Job
+
+Here is an example Scheduled Job. Every minute, it runs a simple job to compute π to 2000 places and prints it out.
+
+{% include code.html language="yaml" file="sj.yaml" ghlink="/docs/user-guide/sj.yaml" %}
+
+Run the example scheduled job by downloading the example file and then running this command:
+
+```shell
+$ kubectl create -f ./sj.yaml
+scheduledjob "pi" created
+```
+
+Alternatively, use `kubectl run` to create a scheduled job without writing full config:
+
+```shell
+$ kubectl run pi --schedule="0/1 * * * ?" --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'
+scheduledjob "pi" created
+```
+
+After creating the scheduled job, get its status using this command:
+
+```shell
+$ kubectl get scheduledjob pi
+NAME      SCHEDULE      SUSPEND   ACTIVE    LAST-SCHEDULE
+pi        0/1 * * * ?   False     0         <none>
+```
+
+As you can see above, there's no active job yet, and no job has been scheduled, either. 
+
+Watch for the job to be created in around one minute:
+
+```shell
+$ kubectl get jobs --watch
+NAME            DESIRED   SUCCESSFUL   AGE
+pi-3972638982   1         1            41s
+```
+
+Now you've seen one running job scheduled by "pi". We can stop watching it and get the scheduled job again:
+
+```shell
+NAME      SCHEDULE      SUSPEND   ACTIVE    LAST-SCHEDULE
+pi        0/1 * * * ?   False     1         Fri, 26 Aug 2016 15:13:00 -0700
+```
+
+## Deleting a Scheduled Job 
+
+Once you don't need a scheduled job anymore, simply delete it with `kubectl`:
+
+```shell
+$ kubectl delete scheduledjob pi
+scheduledjob "pi" deleted
+```
+
+This stops new jobs from being created. However, running jobs won't be stopped, and no jobs or their pods will
+be deleted. To clean up those jobs and pods, you need to list all jobs created by the scheduled job, and delete them all:
+
+```shell
+$ kubectl get jobs
```

Comment:
If you include a label in the jobTemplate then you can filter on that label when listing the jobs, to distinguish from unrelated jobs.